### PR TITLE
Approved Oct 2023 Minutes

### PR DESCRIPTION
**Pull request by hythloda**

### CODE REVIEW

File: Governing Board Public Minutes/2023-10-23.md [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+
+Q: Has A-O reached out to interact and support with new artifact repos to harden against these attacks? Good callout, but have not been proactively reaching out at this point with half the effort so far being half reactive and half proactive.
+
+Q: Are there plans to protect ML repos, like HuggingFace, that let you embed scripts that run? (directed to both OpenSSF and A-O) That will need further discussion outside of this meeting.
+
+Q: Related to constrained funding, how does A-O define success? Depends on the recipient rather than the area of funding. Annual Report will come out soon to show more details.
+
+Briefly to staffing and overall changes to Omega approach. Goal is to catalyze growth and commit to finding scalable approaches. Pivoting from engineering experiment to funding organizations alighed with Omega’s goals
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:549]:
```suggestion
Briefly to staffing and overall changes to Omega approach. Goal is to catalyze growth and commit to finding scalable approaches. Pivoting from engineering experiment to funding organizations aligned with Omega’s goals
```

File: Governing Board Public Minutes/2023-10-23.md [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+    * Question about revenue which is expected to be flat. Current budget exceeds revenue expectations which will have to be adjusted long term.
+
+### Resolving an Identity Crisis: What’s Next for OpenSSF, 2024 Planning and Prioritization
+
+OA presented newly GB approved Mission, Vision, and Strategy,  quotes, and different context types to keep in mind during discussion on future direction of OpenSSF. Types of work currently supported by OpenSSF were outlined in terms of cost, time, and impact. Concern that funding to support security initiatives will decrease as trends change and want to make sure OpenSSF is supporting initiatives that provide higher return and sustainability. The listed criteria are important, but including aspects of sustainability and increased commitment are also important to consider. It was noted that there is dependency between the types of work and want to clarify the duration [of types of work] to better inform the scope. Convening is important to build trust needed for the other types of work, particularly internet-facing services.  \
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:598]:
```suggestion
OA presented newly GB approved Mission, Vision, and Strategy, quotes, and different context types to keep in mind during discussion on future direction of OpenSSF. Types of work currently supported by OpenSSF were outlined in terms of cost, time, and impact. Concern that funding to support security initiatives will decrease as trends change and want to make sure OpenSSF is supporting initiatives that provide higher return and sustainability. The listed criteria are important, but including aspects of sustainability and increased commitment are also important to consider. It was noted that there is dependency between the types of work and want to clarify the duration [of types of work] to better inform the scope. Convening is important to build trust needed for the other types of work, particularly internet-facing services.
```

File: Governing Board Public Minutes/2023-10-23.md [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+
+* Volunteers to determine services model makes sense for OpenSSF: 
+    * Stephen Augustus (Cisco)
+    * Brian Fox (Sonatype) 
+    * Ryan Ware (Intel)
+    * Tim Pepper (VMWare)
+    * Sarah Evans (Dell)
+    * Stephen Walli (Microsoft)
+    * Arun Gupta (Intel)
+    * Anne Bernatio (Google)
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:615]:
```suggestion
    * Anne Bertucio (Google)
```
I think, from the attendee list.

File: Governing Board Public Minutes/2023-10-23.md [hunk 4]
```diff
@@ -0,0 +1,688 @@
...
+CRob gave a top level update on the current state of the TAC, including:
+* Python Software Foundation Alpha Omega work
+    * Idea: How to highlight and reward members for their contributions?
+* Open Source Consumption manifesto
+* Scorecard 4.12 release
+* Security Toolbelt (formerly known as Sterling Toolchain)
+* FSI completing similar effort
+* Sigstore in GA and SLSA 1.0 released
+* Strategic alignment session upcoming in November
+* Goals: WG & Project MVSRs, implementation of consistent processes aligned with approved Operating Model, Security Toolbert v1.0 draft
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:647]:
```suggestion
* Goals: WG & Project MVSRs, implementation of consistent processes aligned with approved Operating Model, Security Toolbelt v1.0 draft
```
?

File: Governing Board Public Minutes/2023-10-23.md [hunk 5]
```diff
@@ -0,0 +1,688 @@
...
+
+SE outlined processes required related to sustainable funding of TIs. TAC does not want to approve funds, but is open to approving TI fund requests from a technical perspective to relay to a budget committee. Comments if that if the amount is under a certain amount, then a separate approval shouldn’t be needed. Defining the process and whether an operating committee requires data on the quantity and types of funding requests tobe useful. Is the 8% allotted for TI funding enough? OpenSSF’s sales pitch is not as strong as other organizations on why projects should choose to join this foundation. Must be able to provide a clear list of benefits projects get when joining OpenSSF. Projects should also be aware of what funding is available for projects and what the process is to request funding. GB needs to determine whether we would like preset amounts per project and/or an operating committee that approves requests. Need to understand where the responsibility and accountability ultimately lies on distributed funds.
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:659]:
```suggestion
SE outlined processes required related to sustainable funding of TIs. TAC does not want to approve funds, but is open to approving TI fund requests from a technical perspective to relay to a budget committee. Comments if that if the amount is under a certain amount, then a separate approval shouldn’t be needed. Defining the process and whether an operating committee requires data on the quantity and types of funding requests to be useful. Is the 8% allotted for TI funding enough? OpenSSF’s sales pitch is not as strong as other organizations on why projects should choose to join this foundation. Must be able to provide a clear list of benefits projects get when joining OpenSSF. Projects should also be aware of what funding is available for projects and what the process is to request funding. GB needs to determine whether we would like preset amounts per project and/or an operating committee that approves requests. Need to understand where the responsibility and accountability ultimately lies on distributed funds. 
```

**Review by joshuagl — COMMENTED** [review]
I was reading through and noticed a few typos.

File: Governing Board Public Minutes/2023-10-23.md [hunk 6]
```diff
@@ -0,0 +1,688 @@
...
+## Approval of Minutes
+
+* _WE HEREBY APPROVE: That the minutes of the August 17th, 2023 meeting of the Board of Directors, in the form attached hereto as Exhibit A, are hereby confirmed, approved and adopted. Minutes attached as [Exhibit A](https://drive.google.com/file/d/18vm0z2q7CShJm-z8C2FvNhR5y_KNemv-/view?usp=sharing)._
+    * Stephen Augustus motioned to approve.
+    * Rao seconded the motion. 
+    * All in favor.
+
+## Approval of MVSR
+
+* _WE HEREBY APPROVE: That the Mission, Vision, Values, and Strategy as defined in the attached in [Exhibit B](https://docs.google.com/document/d/1_oLsODuHRv2kYYn8meBUc9T6e68BZQFsknYBG642acw/edit#heading=h.e9uhthj01hdc) is approved._
```
Comment (joshuagl) [inline Governing Board Public Minutes/2023-10-23.md:534]: I'm not able to access the document here, should it be publcily viewable?

Comment (hythloda) [reply Governing Board Public Minutes/2023-10-23.md:534]: They are now publicly on openssf.org. This is the doc they board viewed, and they have access to the doc, so for historical purposes, we keep the link here.

File: Governing Board Public Minutes/2023-10-23.md (hunk 2 above)
Comment (hythloda) [reply Governing Board Public Minutes/2023-10-23.md:598]: This is from the automated docs to md conversion we use.

File: Governing Board Public Minutes/2023-10-23.md (hunk 3 above)
Comment (hythloda) [reply Governing Board Public Minutes/2023-10-23.md:615]: correct. thanks!

**Pull request merged**